packages/web-components/src/components/cbp-toast/cbp-toast.stories.tsx

```diff
@@ -0,0 +1,107 @@
+    return ` 
+          <cbp-toast
+            color=${color}
+            timer=${timer}
+            ${icon ? `icon=${icon}` : ''}
+            ${context && context != 'light-inverts' ? `context=${context}` : ''}
+            ${sx ? `sx=${JSON.stringify(sx)}` : ''}
+          >
+            <div slot="cbp-toast-title"> Test Toast Title</div>
+            ${content}
+            <div slot="cbp-toast-buttons">${buttons}</div>
+          </cbp-toast>
+        `;
+  };
+  
+  export const Toast = Template.bind({});
+  
+  Toast.args = {
+    icon: 'user',
+    content: 'Notification Description - A rule you are following just fired.',
+    buttons: '<cbp-button type="button" fill="ghost" color="secondary"> Default </cbp-button> <cbp-button type="button" fill="ghost" color="secondary"> Default 2</cbp-button>'
+  }
+
+  const MultiTemplate = ({ icon, content, buttons, timer, color, context, sx }) => {
+    return ` 
+          <cbp-toast
+            color=${color}
+            timer=${timer}
+            ${icon ? `icon=${icon}` : ''}
+            ${context && context != 'light-inverts' ? `context=${context}` : ''}
+            ${sx ? `sx=${JSON.stringify(sx)}` : ''}
+          >
+            <div slot="cbp-toast-title"> Test Toast Title</div>
+            ${content}
+            <div slot="cbp-toast-buttons">${buttons}</div>
+          </cbp-toast>
+        
+          <cbp-toast
+          color=${color}
+          timer=${timer}
+          ${icon ? `icon=${icon}` : ''}
+          ${context && context != 'light-inverts' ? `context=${context}` : ''}
+          ${sx ? `sx=${JSON.stringify(sx)}` : ''}
+        >
+          <div slot="cbp-toast-title"> Test Toast Title</div>
+          ${content}
+          <div slot="cbp-toast-buttons">${buttons}</div>
+        </cbp-toast>
+        
+        <cbp-toast
+        color=${color}
+        timer=${timer}
+        ${icon ? `icon=${icon}` : ''}
+        ${context && context != 'light-inverts' ? `context=${context}` : ''}
+        ${sx ? `sx=${JSON.stringify(sx)}` : ''}
+      >
+        <div slot="cbp-toast-title"> Test Toast Title</div>
+        ${content}
+        <div slot="cbp-toast-buttons">${buttons}</div>
+      </cbp-toast>
+
+      <cbp-toast
+      color=${color}
+      timer=${timer}
+      ${icon ? `icon=${icon}` : ''}
+      ${context && context != 'light-inverts' ? `context=${context}` : ''}
+      ${sx ? `sx=${JSON.stringify(sx)}` : ''}
+    >
+      <div slot="cbp-toast-title"> Test Toast Title</div>
+      ${content}
+      <div slot="cbp-toast-buttons">${buttons}</div>
+    </cbp-toast>
+          `;
+  };
+
+  export const MultipleToast = MultiTemplate.bind({});
+
+  MultipleToast.args = {
+    icon: 'user',
+    content: 'Notification Description - A rule you are following just fired.',
+    buttons: '<cbp-button type="button" fill="ghost" color="secondary"> Default </cbp-button> <cbp-button type="button" fill="ghost" color="secondary"> Default 2</cbp-button>'
```

Comment:
I would just slot the buttons. This isn't providing much value allowing them to edit this block of HTML and you have an issue with context that may need to be passed down to them. (they are not visible in dark mode).